### DISCUSSION TITLE
Being fined for 10 days, pls told me how to fix it~

### DISCUSSION THREAD

**Discussion by Jacktalent** [filecoin-project/lotus#7240] [Storage Provider] [answered]
the node is being fined since August 20th. It will be fined some times every day while each time was fined 0.05FIL or 0.09FIL or 0.15FIL. 

Here is the node:   https://filfox.info/zh/address/f01054527

Help pls~~~

Is it the windowPost take to much time? But the node did not lose any power

**Comment by de-authority** [comment, marked as answer]
Do you have sectors did precommit but did'nt complete provocmmit? I found so many people get collateral loss because improper operation.
and found this a mounth later.

**Reply by Jacktalent** [reply]
I have checked the miner info, there is really some FIL in the precommint. So I need to finish the procommint then the fined will stop, is that right?
very thx

**Reply by de-authority** [reply]
If you provecommit not done after 30 days. your collateral gone. you can do nothing with that.
if you still have some sectors did complete precommit and did not complete provecommit. and not longer than 30days. continue do the provecommit with these sectors.

**Reply by Jacktalent** [reply]
The sectors which completed precommit but not complete provecommit are removed some days ago :( , so the collateral FIL will continue to be fined until 0?

**Reply by de-authority** [reply]
you loss your collateral in that remove sectors

**Reply by Jacktalent** [reply]
cry for the lost collateral :((
thx very much~
